### PR TITLE
RHTAPINST-61: Generate a StackRox API Token

### PR DESCRIPTION
**Pull request by otaviof**
When ACS is installed an API token is generated and stored as a secret.

### CODE REVIEW

File: charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{{- if .Values.backingServices.acs.enabled -}}
+  {{- $acs := .Values.backingServices.acs -}}
+  {{- $name := printf "%s-post-deploy-%d" $acs.name .Release.Revision }}
```
Comment (Roming22) [inline charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml:3]: Why use of the revision #?

**Comment by Roming22** [comment]
I just did a test and the acs tasks of the software template failed. My guess is that something is not right during deployment as I don't have a `rhtap-acs-integration` secret.

File: charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  labels:
+    {{- include "rhtap-backing-services.labels" . | nindent 4 }}
+  namespace: {{ $acs.namespace }}
```
Comment (Roming22) [inline charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml:16]: That job has to be run in the rhtap namespace as the `rhtap-acs-integration` is expected to be there.

File: charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml (hunk 1 above)
Comment (otaviof) [reply charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml:3]: Let's tackle the revision numbers, and the Helm labels to delete/prune jobs in a another PR, as we've spoken.

File: charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml (hunk 2 above)
Comment (otaviof) [reply charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml:16]: Cool, thanks.

**Review by Roming22 — APPROVED** [review]

**Pull request merged**